modules/mlservices/nlp/userdata.tpl

```diff
@@ -62,13 +60,13 @@ write_files:
         WantedBy=multi-user.target
     path: /etc/systemd/system/docker-${service_name}-api.service
     permissions: '0644'
--   content: |
-        [Service]
-        Environment="HTTP_PROXY=http://${proxy_endpoint}" "HTTPS_PROXY=http://${proxy_endpoint}" "NO_PROXY=169.254.169.254,localhost,127.0.0.1,/var/run/docker.sock"
+-   content: |	
+        [Service]	
+        Environment="HTTP_PROXY=http://${proxy_endpoint}" "HTTPS_PROXY=http://${proxy_endpoint}" "NO_PROXY=169.254.169.254,localhost,127.0.0.1,/var/run/docker.sock"	
     path: /etc/systemd/system/docker.service.d/http-proxy.conf
     permissions: '0644'
--   content: |
-        HTTP_PROXY=http://${proxy_endpoint}
-        HTTPS_PROXY=http://${proxy_endpoint}
-        NO_PROXY=169.254.169.254
-    path: /var/awslogs/etc/proxy.conf
+-   content: |	
+        HTTP_PROXY=http://${proxy_endpoint}	
+        HTTPS_PROXY=http://${proxy_endpoint}	
+        NO_PROXY=169.254.169.254	
+    path: /var/awslogs/etc/proxy.conf
```

Comment:
Should we be switching this to sh script with the proxy check?

Reply:
changed all the ML, and the proxy module to sh